### DISCUSSION TITLE
Groovy Parser

### DISCUSSION THREAD

**Discussion by SethGower** [tree-sitter/tree-sitter#1274] [Grammars] [answered]
Hello, I apologize if this isn't the correct place. I was looking for a gitter or something similar to post this question, but I didn't see one. 

I am trying to find (if there exists) a tree-sitter parser for Groovy (specifically for Jenkinsfiles). When I googled it, it was showing the [tree-sitter-java](https://github.com/tree-sitter/tree-sitter-java) parser. I know Groovy is related to java, so I figured this might work with it. I tried parsing a jenkinsfile (in [`nvim-treesitter`](https://github.com/nvim-treesitter/nvim-treesitter)) as a groovy file (`set ft=groovy`) and checking the Playground but it didn't show anything. 

Are there any parsers for groovy/jenkinsfiles?

**Comment by Hubro** [comment]
There doesn't seem to be, unfortunately.

**Comment by ObserverOfTime** [comment, marked as answer]
https://github.com/murtaza64/tree-sitter-groovy
